Lib/zipimport.py

```diff
@@ -222,7 +222,7 @@ def load_module(self, fullname):
         Deprecated since Python 3.10. Use exec_module() instead.
         """
         msg = ("zipimport.zipimporter.load_module() is deprecated and slated for "
-               "removal in Python 3.12; use exec_module() instead")
+               "removal in Python 3.15; use exec_module() instead")
```

Comment:
We should probably update this to use `warnings._deprecated()`.

https://github.com/python/cpython/blob/5b7a872b26a9ba6c93d7c2109559a82d1c1612de/Lib/warnings.py#L650-L668